### PR TITLE
Remove async/await

### PR DESCRIPTION
**Pull request by dubiousdavid**
* Remove async/await.
* Fmt with prettier.
* Changelog and badge.

### CODE REVIEW

**Comment by dubiousdavid** [comment]
Note: The code changes here are untested.

File: README.md [hunk 1]
```diff
@@ -41,8 +47,8 @@ let LinkedInButton = ({ onClick }) => (
   </Button>
 )
 
-let onSignIn = (authPayload) => {  
-  // Use the authentication payload to verify 
+let onSignIn = (authPayload) => {
+  // Use the authentication payload to verify
   // the identity of the request using server
   // side authentication procedures.
   console.log(authPayload)
```
Comment (geosp) [inline README.md]: Can we remove this console log?

File: src/util/google.js [hunk 2]
```diff
@@ -3,45 +3,47 @@ import { loadScript, hasRequiredSettings } from './common'
...
-      await load('auth2')
-      if (!window.gapi.auth2.getAuthInstance()) {
-        window.gapi.auth2.init({
-          client_id: _.trimEnd(appId, '.apps.googleusercontent.com'),
-          fetch_basic_profile: fetchBasicProfile,
-          scope,
-        })
-      }
-  }
+      load('auth2').then(() => {
```
Comment (daedalus28) [inline src/util/google.js]: dont forget to return

File: src/util/google.js [hunk 3]
```diff
@@ -3,45 +3,47 @@ import { loadScript, hasRequiredSettings } from './common'
...
+    }
+  )
+}
 
-  export let onClick = async ({appId, onSuccess}) => {
-    let auth2 = window.gapi.auth2.getAuthInstance()
-    let user = await auth2.signIn()
+export let onClick = ({ appId, onSuccess }) => {
+  let auth2 = window.gapi.auth2.getAuthInstance()
+  auth2.signIn().then(user => {
```
Comment (daedalus28) [inline src/util/google.js]: dont forget to return

**Pull request merged**